builder_data.yaml

```diff
@@ -50,10 +50,11 @@ apt_repo_command: |
   keychain -k mine;
 yum_repo_command: |
   for repodir in $(find "__REPO_PATH__" -mindepth 2 -name "*.rpm" | xargs -I {} dirname {} | uniq) ; do
-    [ -d "${repodir}" ] || continue;
-    echo "Generating repodata for ${repodir}..."
-    sudo createrepo --checksum=sha --database --update "${repodir}";
-    echo "Finished generating repodata for ${repodir}..."
+    [ -d "${repodir}" ] || continue &&
+    echo "Generating and signing repodata for ${repodir} with __GPG_KEY__..." &&
+    sudo createrepo --checksum=sha --database --update "${repodir}" &&
+    gpg --armor --detach-sign -u __GPG_KEY__ "${repodir}/repodata/repomd.xml" &&
+    gpg --export --armour __GPG_KEY__ > "${repodir}/repodata/repomd.xml.key"
```

Comment:
I don't think you can export without subkeys; in theory I believe the workflow is exporting the subkey itself and using *that* to sign stuff. We didn't do that, so I wrote a script which strips subkeys a while back. I'll have to hunt to find it though -- it's on an external drive somewhere.